### PR TITLE
14.0 mig product_configurator_stock

### PR DESCRIPTION
**Pull request by pledrateam**

### CODE REVIEW

**Comment by PCatinean** [comment]
PR ready for review @dreispt

File: product_configurator_stock/README.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+===========================
```
Comment (dreispt) [inline product_configurator_stock/README.rst]: Can you please add a readme/DESCRIPTION.rst file?

File: product_configurator_stock/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "data": [
+        "data/menu_product.xml",
+        "security/ir.model.access.csv",
+        "views/stock_picking_view.xml",
+        "views/stock_move_view.xml",
+        "views/product_view.xml",
+    ],
+    "demo": ["demo/product_template.xml"],
+    "images": [],
+    "test": [],
```
Comment (dreispt) [inline product_configurator_stock/__manifest__.py]:
```suggestion
```

File: product_configurator_stock/README.rst (hunk 1 above)
Comment (dreispt) [reply product_configurator_stock/README.rst]: Reading further, some basic USAGE.rst instructions would be much appreciated.

**Review by dreispt — COMMENTED** [review]
Code review LGTM, added a few minor comments.

File: product_configurator_stock/__init__.py [hunk 3]
```diff
@@ -0,0 +1,2 @@
+from . import models
```
Comment (patrickrwilson) [inline product_configurator_stock/__init__.py:3]: Please add license info at top.

File: product_configurator_stock/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (patrickrwilson) [inline product_configurator_stock/__manifest__.py:3]: Please add license info at top.

File: product_configurator_stock/data/menu_product.xml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8" ?>
```
Comment (patrickrwilson) [inline product_configurator_stock/data/menu_product.xml]: This line is no longer needed. (Not a blocker, just cosmetic)

File: product_configurator_stock/demo/product_template.xml [hunk 6]
```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="utf-8" ?>
```
Comment (patrickrwilson) [inline product_configurator_stock/demo/product_template.xml]: This line is no longer needed. (Not a blocker, just cosmetic)

File: product_configurator_stock/demo/product_template.xml [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+<openerp>
```
Comment (patrickrwilson) [inline product_configurator_stock/demo/product_template.xml]: Rename tag to 'odoo'

File: product_configurator_stock/demo/product_template.xml [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+<data>
```
Comment (patrickrwilson) [inline product_configurator_stock/demo/product_template.xml:2]: remove 'data' tag

File: product_configurator_stock/models/stock_move.py [hunk 9]
```diff
@@ -0,0 +1,43 @@
+from odoo import fields, models
```
Comment (patrickrwilson) [inline product_configurator_stock/models/stock_move.py:1]: Please add license info at top.

File: product_configurator_stock/models/__init__.py [hunk 10]
```diff
@@ -0,0 +1,2 @@
+from . import stock_picking
```
Comment (patrickrwilson) [inline product_configurator_stock/models/__init__.py:1]: Please add license info at top.

File: product_configurator_stock/models/stock_picking.py [hunk 11]
```diff
@@ -0,0 +1,27 @@
+from odoo import fields, models
```
Comment (patrickrwilson) [inline product_configurator_stock/models/stock_picking.py:1]: Please add license info at top.

File: product_configurator_stock/views/product_view.xml [hunk 12]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (patrickrwilson) [inline product_configurator_stock/views/product_view.xml]: This line is no longer needed. (Not a blocker, just cosmetic)

File: product_configurator_stock/views/stock_move_view.xml [hunk 13]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (patrickrwilson) [inline product_configurator_stock/views/stock_move_view.xml]: This line is no longer needed. (Not a blocker, just cosmetic)

File: product_configurator_stock/views/stock_picking_view.xml [hunk 14]
```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (patrickrwilson) [inline product_configurator_stock/views/stock_picking_view.xml]: This line is no longer needed. (Not a blocker, just cosmetic)

File: product_configurator_stock/wizard/__init__.py [hunk 15]
```diff
@@ -0,0 +1 @@
+from . import product_configurator
```
Comment (patrickrwilson) [inline product_configurator_stock/wizard/__init__.py:3]: Please add license info at top.

File: product_configurator_stock/wizard/product_configurator.py [hunk 16]
```diff
@@ -0,0 +1,57 @@
+from odoo import fields, models
```
Comment (patrickrwilson) [inline product_configurator_stock/wizard/product_configurator.py:3]: Same, need license info.

**Review by patrickrwilson — CHANGES_REQUESTED** [review]
Just a bunch of small changes (remove unneeded xml lines and add licensing info)

**Comment by PCatinean** [comment]
Changes from review are done

**Review by patrickrwilson — APPROVED** [review]
LGTM

File: product_configurator_stock/views/stock_move_view.xml (hunk 13 above)
Comment (rousseldenis) [reply product_configurator_stock/views/stock_move_view.xml]: @patrickrwilson This encoding is correct for xml. The removal is only for python files with python >=3

**Review by PCatinean — APPROVED** [review]

**Pull request merged**